ts/src/gate.ts

```diff
@@ -1929,10 +1927,13 @@ export default class gate extends Exchange {
          * @see https://www.gate.io/docs/developers/apiv4/en/#generate-currency-deposit-address
          * @param {string} code unified currency code
          * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @param {string} [params.network] unified network code (not used directly by gate.io but used by ccxt to filter the response)
          * @returns {object} an [address structure]{@link https://docs.ccxt.com/#/?id=address-structure}
          */
         await this.loadMarkets ();
         const currency = this.currency (code);
+        const rawNetwork = this.safeStringUpper (params, 'network');
+        params = this.omit (params, 'network');
```

Comment:
@NeOMakinG I think this can break our network mappings. Did you check this part?